views/default/tab_client_sites.pdt

```diff
@@ -0,0 +1,62 @@
+<div class="title_row first">
+    <h3>
+        <?php $this->_('CWatch.tab_sites.sites'); ?>
+        <a href="<?php echo $this->base_uri . 'services/manage/' . $this->Html->ifSet($service->id) . '/tabClientSites?action=add_site';?>" class="btn btn-default pull-right"><?php $this->_('CWatch.tab_sites.add');?></a>
+    </h3>
+</div>
+<table class="table table-bordered" id="sites">
+    <tr>
+        <td><?php $this->_('CWatch.tab_sites.name'); ?></td>
+        <td><?php $this->_('CWatch.tab_sites.licenseKey'); ?></td>
+        <td><?php $this->_('CWatch.tab_sites.status'); ?></td>
+        <td><?php $this->_('CWatch.tab_sites.malware_scanner'); ?></td>
+        <td><?php $this->_('CWatch.tab_sites.actions'); ?></td>
+    </tr>
+    <?php
+    if (!empty($this->Html->ifSet($sites))) {
```

Comment:
php's empty() construct cannot accept anything except variables < php 5.5. This should work on php 5.4 so the Html::ifSet call can be omitted.